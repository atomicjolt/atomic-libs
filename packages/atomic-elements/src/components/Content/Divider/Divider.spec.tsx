import { describe, test, expect } from "vitest";
import { render } from "@testing-library/react";
import { Divider } from ".";

describe("Divider", () => {
  test("renders correctly horizontally", () => {
    const res = render(<Divider />);
    expect(res.asFragment()).toMatchSnapshot();
  });

  test("renders correctly vertically", () => {
    const res = render(<Divider orientation="vertical" />);
    expect(res.asFragment()).toMatchSnapshot();
  });
});
