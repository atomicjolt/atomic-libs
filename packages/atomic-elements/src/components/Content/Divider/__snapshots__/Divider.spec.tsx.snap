// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Divider > renders correctly horizontally 1`] = `
<DocumentFragment>
  <div
    aria-orientation="horizontal"
    class="sc-FEMpB kojJGJ aje-divider"
    role="separator"
  />
</DocumentFragment>
`;

exports[`Divider > renders correctly vertically 1`] = `
<DocumentFragment>
  <div
    aria-orientation="vertical"
    class="sc-FEMpB kojJGJ aje-divider"
    role="separator"
  />
</DocumentFragment>
`;
